lib/brewdler/brew_installer.rb

```diff
@@ -2,14 +2,14 @@ module Brewdler
   class BrewInstaller
     def self.install(name, options = {})
       if `which brew`; $?.success?
-        command = "brew install #{name}"
+        command = [name]
         unless options[:args].nil?
           options[:args].each do |arg|
-            command << " --#{arg}"
+            command << "--#{arg}"
           end
         end
 
-        `#{command}`
+        system "brew", "install", *command
```

Comment:
I think wouldn't be better to print out current state. So if I have a long list in the `Brewfile`, I can know the current progress.